doc/source/user_guide/tutorials/mesh/create_a_mesh_from_scratch.rst

```diff
@@ -0,0 +1,158 @@
+.. _ref_tutorials_create_a_mesh_from_scratch:
+
+==========================
+Create a mesh from scratch
+==========================
+
+.. include:: ../../../links_and_refs.rst
+
+This tutorial demonstrates how to build a |MeshedRegion| from the scratch.
+
+The mesh object in DPF is a |MeshedRegion|. You can create your own |MeshedRegion| object and use it
+with DPF operators. The ability to use scripting to create any DPF entity means
+that you are not dependent on result files and can connect the DPF environment
+with any Python tool.
+
+In this tutorial, we create a parallel piped mesh made of linear hexa elements.
+
+:jupyter-download-script:`Download tutorial as Python script<create_a_mesh_from_scratch>`
+:jupyter-download-notebook:`Download tutorial as Jupyter notebook<create_a_mesh_from_scratch>`
+
+Import the necessary modules
+----------------------------
+
+Import the ``ansys.dpf.core`` module, including the operators module and the numpy library.
+
+.. jupyter-execute::
+
+    # Import the numpy library
+    import numpy as np
+    # Import the ``ansys.dpf.core`` module
+    from ansys.dpf import core as dpf
+    # Import the operators module
+    from ansys.dpf.core import operators as ops
+
+Define the mesh dimensions
+--------------------------
+
+.. jupyter-execute::
+
+    # Define the mesh dimensions
+    length = 0.1
+    width = 0.05
+    depth = 0.1
```

Comment:
What about this just to short a bit the code and use the horizontal space in the body of the docs?

```suggestion
    # Define the mesh dimensions
    length, width, depth = 0.1, 0.05, 0.1
```